tests/python_client/base/utility_wrapper.py

```diff
@@ -16,6 +16,59 @@ class ApiUtilityWrapper:
 
     ut = utility
 
+    def bulk_load(self, collection_name,  partition_name="",
+                  channels="", row_based=True, files="", timeout=None,
+                  using="default", check_task=None, check_items=None, **kwargs):
+        func_name = sys._getframe().f_code.co_name
+        res, is_succ = api_request([self.ut.bulk_load, collection_name, partition_name,
+                                    channels, row_based,files, timeout,
+                                    using], **kwargs)
+        check_result = ResponseChecker(res, func_name, check_task, check_items, is_succ,
+                                       collection_name=collection_name, using=using).run()
+        return res, check_result
+
+    def get_bulk_load_state(self, task_id, timeout=None, using="default", check_task=None, check_items=None,  **kwargs):
+        func_name = sys._getframe().f_code.co_name
+        res, is_succ = api_request([self.ut.get_bulk_load_state, task_id, timeout, using], **kwargs)
+        check_result = ResponseChecker(res, func_name, check_task, check_items, is_succ,
+                                       task_id=task_id, using=using).run()
+        return res, check_result
+
+    def wait_for_bulk_load_tasks_completed(self, task_ids, timeout=None, using="default", **kwargs):
+        start = time.time()
+        successes = {}
+        fails = {}
+        if timeout is not None:
+            task_timeout = timeout / len(task_ids)
+        else:
+            task_timeout = TIMEOUT
+        while True and (len(successes) + len(fails)) < len(task_ids):
```

Comment:
Here what is the function of "True"?

Reply:
to make sure there are only 2 ways to stop waiting for completed:
1. all tasks result in success or fail
2. timeout